CMake/kwiver-depends-DBoW2.cmake

```diff
@@ -0,0 +1,6 @@
+# Optional confgure DBoW2 dependency
+
+option( KWIVER_ENABLE_DBOW2
+  "Enable DBoW2 dependent code and plugins"
+  ON
+  )
```

Comment:
Please add a section that includes `find_package( OpenCV REQUIRED )` if dbow2 is enabled. It should be similar to https://github.com/Kitware/kwiver/blob/master/CMake/kwiver-depends-OpenCV.cmake#L9-L10.